src/main/java/wile/engineersdecor/blocks/EdHatchBlock.java

```diff
@@ -58,18 +57,16 @@ public EdHatchBlock(long config, BlockBehaviour.Properties builder, final AABB u
   public EdHatchBlock(long config, BlockBehaviour.Properties builder, final AABB[] unrotatedAABBsClosed, final AABB[] unrotatedAABBsOpen)
   { super(config, builder, unrotatedAABBsClosed); vshapes_open = makeHorizontalShapeLookup(unrotatedAABBsOpen); }
 
-  protected static ArrayList<VoxelShape> makeHorizontalShapeLookup(final AABB[] unrotatedAABBs)
+  protected static List<VoxelShape> makeHorizontalShapeLookup(final AABB[] unrotatedAABBs)
   {
-    return new ArrayList<>(Arrays.asList(
+    return List.of(
```

Comment:
Java 11 addition

Reply:
Japp, Java supports that now, good point.